tests/loading_from_clipboard/loading_from_clipboard_test.cpp

```diff
@@ -114,12 +123,142 @@ namespace {
         DecklistBuilder decklistBuilder = DecklistBuilder();
         deckList->forEachCard(decklistBuilder);
 
-        CardRows expectedMainboard = CardRows({{"CardThatDoesNotExistInCardsXml", 1}});
+        CardRows expectedMainboard = CardRows({
+                                                      {"cardthatdoesnotexistincardsxml", 1}
```

Comment:
That's only a temporary fix, there's probably a CLion configuration that I can edit just not sure which